README.md

```diff
@@ -104,13 +104,13 @@ This should make the approach generally fast and small. Actual science and optim
 Anything coercible to strings are supported:
 
 ```javascript
-const render = () => html`foo is ${foo}`;
```

Comment:
Our style is to always use parens for arrow functions.

Reply:
ok but write argument name in parens

Reply:
(foo) => ...

Reply:
These aren't necessarily parameters, they could be closed over variables from outer scopes.

Reply:
But the user should guess this